pkg/addons/adminconsole/adminconsole.go

```diff
@@ -183,6 +184,16 @@ func (a *AdminConsole) GenerateHelmConfig(onlyDefaults bool) ([]v1beta1.Chart, [
 		} else {
 			helmValues["isAirgap"] = "false"
 		}
+		if len(a.proxyEnv) > 0 {
+			extraEnv := []map[string]interface{}{}
+			for k, v := range a.proxyEnv {
+				extraEnv = append(extraEnv, map[string]interface{}{
+					"name":  k,
+					"value": v,
+				})
+			}
+			helmValues["extraEnv"] = extraEnv
```

Comment:
don't kill me, but this will be wiped on upgrade/helm reapply

you'll likely need to add the proxy values to the installation object and then insert them properly in operator

Reply:
this isn't a blocker here, since we only really need it on initial install/restore, but if when we eventually want to make proxy a supported flag we'll need to handle it